diff -ruN ntfs-3g-0.20061212-BETA./src/Makefile.am ntfs-3g-0.20061212-BETA/src/Makefile.am
--- ntfs-3g-0.20061212-BETA./src/Makefile.am	2006-12-09 00:47:08.000000000 +0100
+++ ntfs-3g-0.20061212-BETA/src/Makefile.am	2006-12-12 16:35:27.909571124 +0100
@@ -38,7 +38,6 @@
 install-exec-hook:
 	$(INSTALL) -d $(DESTDIR)/sbin
 	$(LN_S) -f $(bindir)/ntfs-3g $(DESTDIR)/sbin/mount.ntfs-3g
-	$(LDCONFIG)
 
 install-data-hook:
 	$(INSTALL) -d $(DESTDIR)$(man8dir)
